deploy/helm/flightctl/charts/keycloak/templates/keycloak-ingress.yaml

```diff
@@ -1,30 +0,0 @@
-{{- if .Values.global.flightctl.useIngress -}}
```

Comment:
I didn't notice we were killing the ingresses, can we keep them for now?

We will need to polish them to be useful anyway, but...

Reply:
We dont have ingress for anything else (like FCTL API)
I'd rather clean them up and then reintroduce the ingress vs route option in one coherent PR.

WDYT ?

Reply:
let's re-add later at once, we can also try to make it work with gateway api/on a podman kind env, which would be super cool for demos.